source/renderer/app/containers/wallet/dialogs/WalletImportDialogContainer.js

```diff
@@ -44,82 +35,19 @@ export default class WalletImportDialogContainer extends Component<
     this.props.actions.dialogs.closeActiveDialog.trigger();
   };
 
-  onWalletNameChange = (params: { index: number, name: string }) => {
-    this.props.actions.walletMigration.updateWalletName.trigger(params);
-  };
-
-  onToggleWalletImportSelection = (params: { index: number }) => {
-    this.props.actions.walletMigration.toggleWalletImportSelection.trigger(
-      params
-    );
-  };
-
-  onSelectExportSourcePath = (params: { importFrom: ImportFromOption }) => {
-    this.props.actions.walletMigration.selectExportSourcePath.trigger(params);
-  };
-
-  onResetExportSourcePath = () => {
-    this.props.actions.walletMigration.resetExportSourcePath.trigger();
-  };
-
   render() {
-    const { app, walletMigration } = this.props.stores;
-    const {
-      exportedWallets,
-      exportErrors,
-      exportSourcePath,
-      defaultExportSourcePath,
-      pendingImportWalletsCount,
-      isExportRunning,
-      isRestorationRunning,
-      walletMigrationStep,
-    } = walletMigration;
-    const { openExternalLink } = app;
-    let walletsCount =
-      this.state.existingWalletsCount + pendingImportWalletsCount;
-    map(exportedWallets, wallet => {
-      if (
-        wallet.import.status === WalletImportStatuses.COMPLETED ||
-        wallet.import.status === WalletImportStatuses.RUNNING
-      ) {
-        walletsCount++;
-      }
-    });
-    const isMaxNumberOfWalletsReached =
-      walletsCount >= MAX_ADA_WALLETS_COUNT;
+    const { stores, actions } = this.props;
+    const { walletMigration } = stores;
+    const { walletMigrationStep } = walletMigration;
+    const { nextStep, resetMigration } = actions.walletMigration;
+    const stepId = IMPORT_WALLET_STEPS[walletMigrationStep];
```

Comment:
@DeeJayElly this is not step ID it is just `step` or `migrationStep`

Reply:
Agree, fixed.